webapi/src/main/twirl/queries/sparql/v1/addValueVersion.scala.txt

```diff
@@ -171,7 +171,7 @@ DELETE {
 
             case uriValue: UriValueV1 => {
 
-                ?newValue knora-base:valueHasUri """@uriValue.uri""" .
+                ?newValue knora-base:valueHasUri """@uriValue.uri"""^^xsd:anyURI .
```

Comment:
this has to be an xsd:anyURI according to knora-base:

```
###  http://www.knora.org/ontology/knora-base#valueHasUri

:valueHasUri rdf:type owl:DatatypeProperty ;

                 rdfs:subPropertyOf :valueHas ;

                 :subjectClassConstraint :UriBase ;

                 :objectDatatypeConstraint xsd:anyURI .

```

@benjamingeer Did we just forget about that?

Reply:
Apparently.